.github/workflows/pr-commands.yaml

```diff
@@ -0,0 +1,55 @@
+on:
+  issue_comment:
+    types: [created]
+name: Commands
+jobs:
+  document:
+    if: startsWith(github.event.comment.body, '/document')
+    name: document
```

Comment:
We can now document the package-code (i.e. the code in R/, tests/ and so on) by starting a PR-comment on GitHub with "/document"